src/shared/components/show-hide/ShowHide.scss

```diff
@@ -1,15 +1,15 @@
 @import 'src/styles/common';
 
 .showHide {
-  // increase specificity to be sure to override chevron's own styles 
+  // increase specificity to be sure to override chevron's own styles
   .chevron {
     margin-left: 10px;
-    height: 6px;
```

Comment:
`cursor: pointer` is missing, if you hoover over chevron for showHide (Entityviewer Filter or Entity Viewer more information), the mouse isn't a pointer.

Reply:
Well spotted 👍 I've pushed an update.

Reply:
Its not properly fixed, now the whole area (even the empty space after the chevron) is clickable (mouse pointer)

Reply:
> Its not properly fixed, now the whole area (even the empty space after the chevron) is clickable (mouse pointer)

It's due to the context where the show/hide is in. If you check out the show/hide itself, it behaves correctly as an inline-block element:

https://user-images.githubusercontent.com/6834224/182147655-aa235ec0-4a9b-4d5d-8e0f-0e0d5aac2f2b.mov

The problem that you are seeing is probably on the entity viewer, where the show/hide expands to fill the css grid column. I'll update the styles there.

Reply:
I pushed an update that should prevent the growing of the show/hide container inside the EntityViewer transcript info panels.